src/TabNine.ts

```diff
@@ -87,8 +91,12 @@ export class TabNine {
   }
 
   private static runTabNine(inheritStdio : boolean = false, additionalArgs: string[] = []): child_process.ChildProcess {
+    const ext = TabNineExtension.getInstance();
```

Comment:
maybe we should create this only once? not for each binary start, this is "static" metadata related to the extension that will not change between the binary instances

Reply:
makes sense